source/_patterns/30-organisms/canton-navigation/canton-navi-collection.scss

```diff
@@ -0,0 +1,117 @@
+@import "../../variables";
+@import "../../mixins";
+
+.canton-nav-collection {
+  box-shadow: $shadow-elevation1;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+
+  @include tablet-up {
+    padding: 24px 32px 24px 24px;
+    flex-direction: row;
+  }
+
+  @include desktop-up {
+    padding: 32px 32px 24px 32px;
+    flex-direction: row;
+  }
+}
+
+/**
+ * Canton navigation content
+ */
+.canton-nav-collection__content {
+  flex-grow:1;
+  flex-shrink: 1;
+  @include tablet-up {
+    flex-basis: 0;
+  }
+}
+
+/**
+ * Kicker
+ */
+.canton-nav-collection__kicker {
+  margin-bottom: 4px;
+}
+
+.canton-nav-collection__kicker--text {
+  font-family: $font-regular;
+  line-height: 1.13;
+  @include fontSize(16);
+  @include font-smoothing();
+  color: $color-accentuation-red;
+
+  @include desktop-up {
+    line-height: 1.11;
+    @include fontSize(18);
+  }
+}
+
+
+/**
+ * Title
+ */
+.canton-nav-collection__title {
+  margin: 0;
+}
+
+.canton-nav-collection__title--text {
+  font-family: $font-medium;
+  font-weight: 500;
+  line-height: $emphasize-line-height-ratio;
+  @include fontSize(20);
+  @include font-smoothing();
+  margin: 0;
+
+  @include desktop-up {
+    @include fontSize(24);
+  }
+}
+
+
+/**
+ * Lead
+ */
+.canton-nav-collection__lead {
+  margin-top: 12px;
+}
+
+.canton-nav-collection__lead--text {
+  font-family: $font-light;
+  line-height: $secondary-line-height-ratio;
+  @include fontSize(16);
+  @include font-smoothing();
+  margin: 0;
+
+  @include desktop-up {
+    @include fontSize(18);
+  }
+}
+
+
+
+/**
+ * Canton switzerland navigation
+ */
+.canton-nav-collection__navigation {
+  display: flex;
+  justify-content: center;
+  flex-grow:1;
+  flex-shrink: 1;
+  margin-top: 24px;
+
+  & > div {
+    margin-left: -24px;
```

Comment:
ouh mist, das ist nur wegen dem doofen Halbkanton links?

Reply:
eventuell eher ein padding-right von 24px auf `.canton-nav` bei smartphone? Nicht besonders viel schöner, aber dann ist's wenigstens in der gleichen Komponente...

Reply:
hab es in die canton-nav verschoben. padding-right klappt im ie11 nicht, daher hab ich es bei margin belassen

Reply:
Ouh, warum nicht die erste Spalte nur halb so breit machen?